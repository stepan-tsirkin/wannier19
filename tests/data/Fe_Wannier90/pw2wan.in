&inputpp
   outdir    = './temp'
   prefix    = 'Fe'
   seedname  = 'Fe'
   write_mmn = .true.
   write_amn = .true.
   write_spn = .true.
   write_sHu = .false.
   write_sIu = .false.
/
