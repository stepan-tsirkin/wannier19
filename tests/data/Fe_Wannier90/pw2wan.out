
     Program PW2WANNIER v.6.7GPU starts on 22Mar2021 at 13: 3:59 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on      16 processor cores
     Number of MPI processes:                16
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =      16
 
  Reading nscf_save data

     Reading xml data from directory:

     ./temp/Fe.save/

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          61      49     17                 1240      884     180
     Max          63      50     18                 1242      887     182
     Sum        1001     797    277                19861    14171    2899
 
     Using Slab Decomposition
 
     Reading collected, re-writing distributed wavefunctions
 
  Spin CASE ( non-collinear )
 
  Wannier mode is: standalone     
 
  -----------------
  *** Reading nnkp 
  -----------------
 
  Checking info from wannier.nnkp file
 
  - Real lattice is ok
  - Reciprocal lattice is ok
  - K-points are ok
  - Number of wannier functions is ok ( 18)
  - All guiding functions are given 
 
 Projections:
    0.000000    0.000000    0.000000 -5  1  1    1.000000
    0.000000    0.000000    0.000000 -5  1  1    1.000000
    0.000000    0.000000    0.000000 -5  2  1    1.000000
    0.000000    0.000000    0.000000 -5  2  1    1.000000
    0.000000    0.000000    0.000000 -5  3  1    1.000000
    0.000000    0.000000    0.000000 -5  3  1    1.000000
    0.000000    0.000000    0.000000 -5  4  1    1.000000
    0.000000    0.000000    0.000000 -5  4  1    1.000000
    0.000000    0.000000    0.000000 -5  5  1    1.000000
    0.000000    0.000000    0.000000 -5  5  1    1.000000
    0.000000    0.000000    0.000000 -5  6  1    1.000000
    0.000000    0.000000    0.000000 -5  6  1    1.000000
    0.000000    0.000000    0.000000  2  2  1    1.000000
    0.000000    0.000000    0.000000  2  2  1    1.000000
    0.000000    0.000000    0.000000  2  3  1    1.000000
    0.000000    0.000000    0.000000  2  3  1    1.000000
    0.000000    0.000000    0.000000  2  5  1    1.000000
    0.000000    0.000000    0.000000  2  5  1    1.000000
 
  Reading data about k-point neighbours 
 
  All neighbours are found 
 
  Opening pp-files 
 
 
  --------------------------
  *** Compute  A projections
  --------------------------
 
  AMN: iknum =       27
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16      17      18      19      20
      21      22      23      24      25      26      27

  AMN calculated
 
  ---------------
  *** Compute  M 
  ---------------
 
  MMN: iknum =       27
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16      17      18      19      20
      21      22      23      24      25      26      27

  MMN calculated
 
  ------------------
  *** Compute  Spin 
  ------------------
 
 iknum =       27
       1
       2
       3
       4
       5
       6
       7
       8
       9
      10
      11
      12
      13
      14
      15
      16
      17
      18
      19
      20
      21
      22
      23
      24
      25
      26
      27
 
  SPIN calculated
 
  -----------------------------------
  *** Orbital terms are not computed 
  -----------------------------------
 
  ----------------
  *** Compute shc 
  ----------------
 
  *** Computing  sHu 
  *** Computing  sIu 
 
 iknum =       27
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16      17      18      19      20
      21      22      23      24      25      26      27 
  shc calculated
 
  ----------------
  *** Write bands 
  ----------------
 
 
  -----------------------------
  *** Plot info is not printed 
  -----------------------------
 
  -----------------------------
  *** Parity info is not printed 
  -----------------------------
 
  ------------
  *** Stop pp 
  ------------
 
 
     init_pw2wan  :      0.31s CPU      0.99s WALL (       1 calls)
     compute_amn  :      1.32s CPU      1.38s WALL (       1 calls)
     compute_mmn  :      1.06s CPU      1.40s WALL (       1 calls)
     compute_shc  :      4.05s CPU      5.00s WALL (       1 calls)
 
     PW2WANNIER   :      6.80s CPU      8.87s WALL

 
   This run was terminated on:  13: 4: 8  22Mar2021            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
 
